Source/WebCore/html/NumberInputType.cpp

```diff
@@ -345,22 +346,22 @@ static bool isE(char16_t ch)
     return ch == 'e' || ch == 'E';
 }
 
-static bool isPlusSign(UChar ch)
+static bool isPlusSign(char16_t ch)
 {
     return ch == '+';
 }
 
-static bool isSignPrefix(UChar ch)
+static bool isSignPrefix(char16_t ch)
 {
     return ch == '+' || ch == '-';
 }
 
-static bool isDigit(UChar ch)
+static bool isDigit(char16_t ch)
 {
     return ch >= '0' && ch <= '9';
 }
```

Comment:
This is a duplicate of the `isASCIIDigit` function from `ASCIICType.h`. Maybe we don’t need this one.